package.json

```diff
@@ -261,7 +261,10 @@
     ],
     "nohoist": [
       "**/webpack-preprocessor/babel-loader",
-      "**/webpack-batteries-included-preprocessor/ts-loader"
+      "**/webpack-batteries-included-preprocessor/ts-loader",
```

Comment:
Why are all these changes in webpack dev server and package json here?

Reply:
It is all connected to the fact that we need `react-scripts` to be resolvable from system tests so that the last step of the installation works.
`react-scripts` loads `jest` and its types and those types conflict with `chai`